### ISSUE TITLE
Resolve ENS domains no matter the URL

### ISSUE THREAD

**Issue by ProgrammerWhoPrograms** [AgregoreWeb/agregore-mobile#34]
Currently, you can't go to a regular HTTPS website that uses an ENS domain. Only an IPFS one.

**Comment by RangerMauve** [comment]
Yeah that would be great. Sadly I don't have active budget for Agregore mobile so if there's a way we could fund this improvement I'd be very much open to it. Maybe in tandem with Handshake or Namecoin?

If anyone has tips on where in the chromium runtime we can hook into DNS I'm all years.
